plugins/automap/src/automap.service.ts

```diff
@@ -304,52 +299,18 @@ export class AutomapService {
     const workbookResponse = await api.workbooks.get(file.workbookId!)
     const sheets = workbookResponse.data.sheets || []
     const { defaultTargetSheet } = this.options
+    const targetSheet =
+      typeof defaultTargetSheet === 'function'
```

Comment:
Do we have a test for this new functionality?

Reply:
There is now